_tags

```diff
@@ -1,5 +1,5 @@
 true : bin_annot, safe_string
-not <ci/**>: warn_error(+1..49), warn(A-3-4-41-44)
+not <ci/**>: warn_error(+1..49-3), warn(A-4-41-44)
```

Comment:
Your change manages to fix your first bullet, but I don't know how to achieve the second point nicely.